webapp/src/hooks/hooks.js

```diff
@@ -35,8 +35,10 @@ export default class Hooks {
                 this.store.dispatch(sendEphemeralPost('You already have a Jira account linked to your Mattermost account. Please use `/jira disconnect` to disconnect.'));
                 return Promise.resolve({});
             }
-            if (getInstalledInstanceType(this.store.getState()) === 'server' && isDesktopApp()) {
-                this.store.dispatch(sendEphemeralPost('Please use your browser to connect to Jira.'));
+
+            if (getInstalledInstanceType(this.store.getState()) === 'server' && isDesktopApp() && !isMinimumDesktopAppVersion(4, 3, 0)) { // eslint-disable-line no-magic-numbers
+                const errMsg = 'The current version of your desktop app does not support connecting to Jira directly. Please connect through your browser or [see here](https://github.com/mattermost/mattermost-plugin-jira) for details on upgrading.';
```

Comment:
```suggestion
                const errMsg = 'Your version of the Mattermost Desktop Client does not support authenticating between Jira and Mattermost directly.  To connect your Jira account with Mattermost, Go to Mattermost via your web browser and then type `/jira connect` or [check the Mattermost download page](https://mattermost.com/download/#mattermostApps) to get the latest version of the desktop client';
```
@mickmister We want them to update their desktop client not the jira plugin (unless I'm missing something).

Reply:
@mickmister is there any way for us to make a link like `[GO HERE](https://<MM_SITEURL>)` so they can just click on the link, instead of them trying to find out where their mattermost server is located?

Reply:
Unfortunately the desktop app opens up `https://<MM_SITEURL>` in a popup since it is the same domain, so it doesn't open the browser. We can display it with `` so it can be used for copy and paste, and not render as a clickable link.

Reply:
@aaronrothschild The original wording of the error was pointing to Jira docs, because I was thinking we would have a full explanation of what's needed to upgrade, including having server `5.16` or other backport-supported server versions. If we can assume they have an up-to-date server version, then we can link to the desktop app download page like you suggest. That seems most appropriate for us too.

Reply:
@aaronrothschild Do you mind taking another look at this when you have a chance?